doc/JSON_FLAGS.md

```diff
@@ -1322,6 +1322,7 @@ Those flags are added by the game code to specific items (that specific welder,
 - ```FIELD_DRESS_FAILED``` Corpse was damaged by unskillful field dressing. Affects butcher results.
 - ```FIELD_DRESS``` Corpse was field dressed. Affects butcher results.
 - ```FIT``` Reduces encumbrance by one.
+- ```FILTHY``` Item always spawns as filthy.
```

Comment:
Also, this doesn't mean the item just spawns as filthy, it means it's permanently filthy.

Reply:
https://github.com/CleverRaven/Cataclysm-DDA/blob/3fab7d5849145a78d9b615dd1e7d2993d8d0b0f3/src/activity_item_handling.cpp#L596
disagrees, unless I'm missing something?

Reply:
You might be thinking this is an item flag set in json, it's not, the surrounding documentation indicates that it's set on the item instance rather than the item type.  Alternately, if that is what is meant by this, it's in the wrong section of the document, this is the section for dynamic flags.

Reply:
Finally, if this IS regarding setting this flag in an item description, that can remain undocumented, because that is not an intended behavior and is not supported.

Reply:
Oh you're right, my bad. I was thinking this was an itype flag.